incubator/hnc/test/e2e/demo_test.go

```diff
@@ -186,7 +188,8 @@ spec:
 			"├── [s] " + nsService1 + "\n" +
 			"├── [s] " + nsService2 + "\n" +
 			"└── [s] " + nsService3
-		RunShouldContain(expected, 2, "kubectl hns tree", nsTeamA)
+		// The subnamespaces takes a bit of time to show up
+		RunShouldContain(expected, 10, "kubectl hns tree", nsTeamA)
```

Comment:
Does this take as much as 10 seconds? I know it's in an `Eventually` that if it passes, it won't take 10 seconds, so basically having a larger number could improve the pass rate for our e2e tests. Just curious that this case could fail with a smaller number than 10?

Nit: can we make the 10 and the 5 seconds below into constants with easy to understand name for different sync time instead of having numbers all over? (A suggestion from Adrian on my conversion test too).

Reply:
Done. And 5s actually has a good pass rate too, I tried for 5 times and they all pass, so I changed the time limit to 5s